lib/glific_web/resolvers/messages.ex

```diff
@@ -239,4 +239,16 @@ defmodule GlificWeb.Resolvers.Messages do
       message -> {:ok, message}
     end
   end
+
+  @doc false
+  @spec send_message_in_wa_group(Absinthe.Resolution.t(), %{input: map()}, %{context: map()}) ::
+          {:ok, any} | {:error, any}
+  def send_message_in_wa_group(_, %{input: params}, %{context: %{current_user: user}}) do
+    case Glific.Providers.Maytapi.Message.create_and_send_wa_message(user, params) do
+      {:ok, _} -> {:ok, true}
+      {:error, error} -> {:error, error}
+    end
+
+    {:ok, true}
```

Comment:
As a FE facing function, shouldn't we provide a better error message if sending failed?